app/src/main/java/com/bakkenbaeck/sol/view/custom/SunView.java

```diff
@@ -0,0 +1,243 @@
+package com.bakkenbaeck.sol.view.custom;
```

Comment:
Isn't this class meant to be in the SunViewLib and not this repo? Might have been checked in accidentally.

Reply:
It was not checked in by accident. 

SunViewLib has 1 fork (my own) and 4 followers. It consists of one view-class (~250 lines of code) and some res-values. The library has not been updated in 2 years and it crashes on the latest versions of Android. It would not be difficult to fix it, but with all this, and especially the library's popularity in mind, I think we should consider :dagger: it and move SunView into daylight permanently. If there were plans to extend or improve the library I think it would make more sense to keep it, but to me it looks very much like it was abandoned a long time ago.